app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+  after_save :update_posts_counter
+  def update_posts_counter
+    author.upadate(posts_counter: author.posts.count)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_posts_counter` should be marked as private after implementing the `after_save` method.